patches/server/0647-Missing-Entity-Behavior-API.patch

```diff
@@ -818,6 +820,26 @@ index 9569068c0a6e1eb934f063634f5d3efe674aa33d..dfd0b5e018194343ca40629db6f70c60
 +    public int getFuseTicks() {
 +        return this.getHandle().getFuse();
 +    }
++
++    @Override
++    public boolean isPrimed() {
++        return this.getHandle().isPrimed();
++    }
++
++    @Override
++    public void primeFuse() {
++        this.getHandle().primeFuse();
++    }
++
++    @Override
++    public void explode() {
++        explode(this.getHandle().getDeltaMovement().horizontalDistanceSqr());
++    }
++
++    @Override
++    public void explode(double power) {
++        this.getHandle().explode(power);
```

Comment:
You need to have a check that it’s nonnegative as its square root is taken inside that method.

Reply:
Nice catch! Added a check for that